Code/Framework/AzToolsFramework/Tests/Prefab/Overrides/PrefabInspectorOverrideTestFixture.cpp

```diff
@@ -28,6 +32,9 @@ namespace UnitTest
     {
         m_testEntityPropertyEditor.reset();
         PrefabOverrideTestFixture::TearDownEditorFixtureImpl();
+        AZ::SettingsRegistryInterface* registry = AZ::SettingsRegistry::Get();
+        registry->Set("/O3DE/Autoexec/ConsoleCommands/ed_enableDPEInspector", m_ed_enableDPEInspector);
+        registry->Set("/O3DE/Autoexec/ConsoleCommands/ed_enableOutlinerOverrideManagement", m_ed_enableOutlinerOverrideManagement);
```

Comment:
We should be consistent with using the AZ::Console here.
```suggestion
        if (auto* console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
        {
            console->PerformCommand("ed_enableOutlinerOverrideManagement", ConsoleCommandContainer{ AZ::ConsoleTypeHelpers::ToString(m_ed_enableDPEInspector) });
            console->PerformCommand("ed_enableDPEInspector", ConsoleCommandContainer{ AZ::ConsoleTypeHelpers::ToString(m_ed_enableOutlinerOverrideManagement) });

        }
```